_src/_posts/2017-09-11-importance-of-serving-ads.markdown

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+title: The importance of serving the right amount of ads 
+date:   2017-09-11
+tag: data-science
+categories:
+ - data-science
+keywords: "ROI, impressions, clicks, advertising, data science, impression frequency cap, CPA, quality, view through attribution"
+description: In this post we analyze how different levels of impressions affect the 
+impact of advertising in multiple campaigns.
+author: jdemonasterio
+---
+
+
+<!--excerpt.start-->
+
+[Jampp] [jampp] helps customers boost their mobile sales with performance marketing. One of the most challenging problems of the industry is the impact quantification an ad has on users. A poorly tackled problem is understanding when a user has been affected enough by ads, and why acquiring clicks is not always best aligned with the advertiser's interests.
+We know that, sometimes, serving more impressions to users will result in CPA increase, since we can not influence their behavior any more. The key question is: how many ads are enough? 
+
+<!--excerpt.end→
+
+##Introduction
+
+Performance advertising is relevant to advertisers that want to have a quantifiable measure of their ROI. This means that the investment in advertising has to reach measurable goals such as increased number of in-app activity or app downloads. Then, the payment is directly measured by the value brought by the _events_ caused by ads. 
+
+RTB ad platforms all enforce value prediction at the user and creative level. These typically take into consideration history of information from the auction itself or from the advertiser's campaigns, but little effort is put on the user's history of impressions. 
+
+Today, advertisers spend millions in marketing budget among different partners. They have to determine who takes the credit for helping the user convert, in other words, which partner is the action _attributed_ to.
```

Comment:
helping the user convert -> making the user convert?

Reply:
Well, that's kind of the crux of this post. Do ads make the user convert or help him convert? How many ads do we need? Do we really have causality? I'm accepting the suggestion but I think we can use another (better) word.